deadbelly/breathEZ
# Ec/feature/render header

### What does this PR do? 
>Clearly and concisely describe the feature. 
#### Header
- adds `h1` title of app to `Header` to render
- downloads `react-router-dom`
  - this is initially for the `Link`
- adds `Link` to `Faq`
  -  this is anticipating a specific route called `Faq`
- adds the `SearchBar` component
- renders the `SearchBar` component in the `Header`

#### SearchBar
- uses hook to give `SearchBar` state and update accordingly
- has state update upon typing in the search bar
- adds submit button
- upon clicking submit button, 
  - anticipates potentially sending state of search to `App` to use api call to render proper data
  - this is up for discussion
- clears search bar upon submission
### Is this a feature or Fix/Refactor?  
- feature
### Problems Encountered & Resulting Solutions  
>State any problems encountered 
- We need to decide whether `SearchBar` or `App` is responsible for handling API requests
### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?
- `src/Components/Header/Header.js`
- `src/Components/SearchBar/SearchBar.js`
### How should this be tested?
- once `App` is created
  - pull down this branch (not to main)
  - observe `App` rendering `Header`
  - observe `Header` rendering the title, a link, and the `SearchBar`
>What steps need to be taken to confirm functionality? 
- none yet, as functionality is dependent on other components' existence
### Steps To Take
>Any questions and possible steps moving forward.
- once `App` is made, styling can occur, as then `Header` will actually render
### This PR is related to which issues:
>Tag issue here: `Closes #<issue_number>`
- This affects #5, but should not close it yet as the `Header` is not complete without the `SearchBar/App` API logic decided
